experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -97,43 +125,65 @@ function held_suarez_forcing!(bl, source, state, diffusive, aux, t::Real)
     k_T           = k_a + (k_s - k_a) * height_factor * cos(φ) ^ 4
     k_v           = k_f * height_factor
 
-    source.ρu += -k_v * ρu
+    # TODO: bottom drag should only be applied in tangential direction
+    source.ρu += -k_v * projection_tangential(bl.orientation, aux, ρu)
     source.ρe += -k_T * ρ * cv_d * (T - T_equil)
 end
 
 function main()
     CLIMA.init()
 
-    FT = Float64
-
-    # DG polynomial order
-    N = 5
-
-    # Domain resolution
-    nelem_horz = 6
-    nelem_vert = 8
-    resolution = (nelem_horz, nelem_vert)
-
-    t0 = FT(0)
-    timeend = FT(60) # 400day
-
-    driver_config = config_heldsuarez(FT, N, resolution)
-    ode_solver_type = CLIMA.ExplicitSolverType(solver_method=LSRK144NiegemannDiehlBusch)
-    solver_config = CLIMA.setup_solver(t0, timeend, driver_config,
-                                       ode_solver_type=ode_solver_type)
-
-    # Set up the filter callback
+    # Driver configuration parameters
+    FT            = Float32           # floating type precision
+    poly_order    = 5                 # discontinuous Galerkin polynomial order
+    n_horz        = 5                 # horizontal element number  
+    n_vert        = 5                 # vertical element number
+    days          = 30                 # experiment day number
+    timestart     = FT(0)             # start time (seconds)
+    timeend       = FT(days*24*60*60) # end time (seconds)
```

Comment:
```suggestion
    timeend       = FT(run_days*day) # end time (seconds)
```
(or something like it: point is to use definition of length of day from PlanetParameters; we do not want to re-type it and have novices have to figure out what this is)